src/averageAges.js

```diff
@@ -36,8 +46,16 @@ function calculateMenAverageAge(people, century) {
  *
  * @return {number}
  */
+
+function getIsMother(people, woman) {
+  return people.some(names => names.mother === woman.name);
+}
+
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = people.filter(withChildren ? person => person.sex === 'f'
```

Comment:
Same situation as in the previous comment.

Reply:
Check other similar cases.

Reply:
![Screenshot_99](https://user-images.githubusercontent.com/110056814/223673962-18237761-f704-4c78-a872-9db396377a2e.png)

Carry out for the main function?
Did i understand correctly?